src/python/com/illinoistriangle/bot/outline_bot.py

```diff
@@ -72,39 +70,81 @@ def lambda_handler(data, context):
   handle_event(slack_event)
 
 
-def handle_event(e):
-  if 'subtype' in e and e['subtype'] == VALUE_TYPE_BOT_MESSAGE:
-    return
+def search_thread_parent_for_urls(channel, thread_ts):
+  """return list of urls from thread's parent post"""
+  #TODO enable scope channels:history
+  response = sc.api_call(
+              'channels.replies',
+              channel=channel,
+              thread_ts=thread_ts,
+              )
+
+  urls = []
+  if response['ok'] == 'true' and 'messages' in response:
+    # from all thread messages, find earliest ts and return the text of that post
+    parent_thread_text = sorted([x['text'] for x in response['messages']], key=lambda x: x['ts'])[0]
+
+    urls = find_urls(parent_thread_text)
+
+  return urls
 
-  if e['type'] != VALUE_TYPE_MESSAGE:
-    return
 
-  if FIELD_TEXT not in e:
-    return
+def event_is_human_thread_reply(event):
+  """return True if the event warrants the bot's response"""
 
-  channel_id = e[FIELD_CHANNEL]
-  if channel_id not in channel_map:
-    return
+  # if a message was posted...
+  if 'type' in event and event['type'] == 'message':
+    # ... by someone other than a bot...
+    if 'subtype' not in event or event['subtype'] != 'bot_message':
+      # ... in a thread...
+      if 'thread_ts' in event:
+        # TODO add channel whitelist
+        # ... in an appropriate channel...
+        if 'channel' in event and event['channel'] in ['C7RLA5HAA', ]:
```

Comment:
use `channel_maps[event['channel']]` to lookup the channel name and check on the name

Reply:
addressed

Reply:
looks like not address yet?

Reply:
whoops... :man_facepalming: